website/docs/sync.md

```diff
@@ -36,11 +59,22 @@ You can install this config with the following command:
 kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/demo/basic/sync.yaml
 ```
 
-Once data is synced into OPA, rules can access the cached data under the `data.inventory` document.
+### Working with SyncSet and Config resources
 
-The `data.inventory` document has the following format:
+* Updating a Config's `syncOnly` field or a SyncSet's `gvks` field should dynamically update what objects are synced.
+* Multiple `SyncSet`s may be defined and those will be reconciled by the Gatekeeper syncset-controller. Notably, the [set union](https://en.wikipedia.org/wiki/Union_(set_theory)) of all SyncSet resources' `gvks` and the [Config](sync#replicating-data-with-config) resource's `syncOnly` will be synced into the data client.
+* A resource will continue to be present in the data client so long as a SyncSet or Config still specifies it under the `gvks` or `syncOnly` field.
+
+### Accessing replicated data
 
+Once data is synced, ConstraintTemplates can access the cached data under the `data.inventory` document.
+
+The `data.inventory` document has the following format:
   * For cluster-scoped objects: `data.inventory.cluster[<groupVersion>][<kind>][<name>]`
      * Example referencing the Gatekeeper namespace: `data.inventory.cluster["v1"].Namespace["gatekeeper"]`
   * For namespace-scoped objects: `data.inventory.namespace[<namespace>][groupVersion][<kind>][<name>]`
      * Example referencing the Gatekeeper pod: `data.inventory.namespace["gatekeeper"]["v1"]["Pod"]["gatekeeper-controller-manager-d4c98b788-j7d92"]`
+
+### Auditing From Cache
+
+The [audit](audit.md) feature does not require replication by default. However, when the ``audit-from-cache`` flag is set to true, the audit informer cache will be used as the source-of-truth for audit queries; thus, an object must first be cached before it can be audited for constraint violations. Kubernetes data can be replicated into the audit cache via one of the resources above.
```

Comment:
```suggestion
The [audit](audit.md) feature does not require replication by default. However, when the `audit-from-cache` flag is set to true, the audit informer cache will be used as the source-of-truth for audit queries; thus, an object must first be cached before it can be audited for constraint violations. Kubernetes data can be replicated into the audit cache via one of the resources above.
```